HOWTO.md

```diff
@@ -0,0 +1,135 @@
+#1. Overview
+In this document, we're going to focus on how to encode and decode OpenLR locations 
+on digital maps using modules in this package.
+
+#2. SetUp
+We'll need to add the following dependencies to our pom.xml
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>encoder</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>decoder</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>map</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>data</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>binary</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>xml</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+
+```xml
+<dependency>
+ <groupId>org.openlr</groupId>
+ <artifactId>proto</artifactId>
+ <version>${openlr.version}</version>
+</dependency>
+```
+
+#3. Map Implementation
+
+**Implement the following interfaces against the map you want to use**
+
+<ul>
+<li>openlr.map.Line</li>
+<li>openlr.map.Node</li>
+<li>openlr.map.MapDatabase</li>
+<li>openlr.map.loader.MapLoadParameter</li>
```

Comment:
I don't think the last 2 are strictly needed.

Reply:
yeah but it is there in the map module also the mapdatabase interface doesn't have a loader.
This would make it easier for any who tries to create their own map implementation.